rpc/proto/mayastor.proto

```diff
@@ -64,6 +64,9 @@ service Mayastor {
 
   // Snapshot operations
   rpc CreateSnapshot (CreateSnapshotRequest) returns (CreateSnapshotReply) {}
+
+  // Enumerate block devices on current host
+  rpc GetHostBlockDevices (GetHostBlockDevicesRequest) returns (GetHostBlockDevicesReply) {}
```

Comment:
Minor issue related to consistent names: isn't the "Host" word in method and struct names a bit useless? All methods that are defined in this file deal with objects that are on the Host so that is implicitly assumed. Just like we don't have ListHostReplicas. Shorter method and struct names -> shorter code :-)

On the same note according to rules used in this file so far: "get" means give me one object. "list" means give me an array of objects. So more appropriate name for this method would be "ListBlockDevices" (and similar name change for parameters).

Reply:
Will change these